package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "codepirates-ah-frontend",
+  "version": "1.0.0",
+  "description": "A blogging platform",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run build && babel-node server/app.js",
+    "build": "babel-node ./node_modules/webpack/bin/webpack --env.environment=production --config webpack.config.js",
```

Comment:
This can work. good job though I would prefer that you compile using web pack. I want the server to be able to run by running ``` node server/app.js```

Reply:
Thanks, let me work on this

Reply:
Done